packages/react-tinacms-github/README.md

```diff
@@ -2,6 +2,24 @@
 
 ## Implementation:
 
+## Register the GithubClient 
+
+We will want to use the GithubClient to load/save our content using the Github API. Let's add it as an API plugin.
+
+```ts
+import { TinaCMS } from 'tinacms'
+import { GithubClient } from 'react-tinacms-github'
+
+const cms = new TinaCMS({
+  apis: {
+      github: new GithubClient('/api/proxy-github', process.env.REPO_FULL_NAME),
```

Comment:
An example of what this would look like would be good, or maybe even a hard-coded string.

This makes me think it should be:

```
REPO_FULL_NAME=tinacms/tinacms.org
```

Am I correct?